plugins/view-resources/src/components/ActionHandler.svelte

```diff
@@ -131,8 +131,11 @@
     const targetTagName = (evt.target as any)?.tagName?.toLowerCase()
 
     let elm = evt.target as HTMLElement
+
+    const isTextEditor = elm.classList.contains('text-editor-view')
```

Comment:
I do not like to bind class here, I think better to be to have some actions allowed per content editable and mark Cmd+K as it.